tools/reassign-alias.js

```diff
@@ -0,0 +1,43 @@
+const elastic = require('../lib/elastic')
+
+function displayErrorMessage (value) {
+  console.log(`You have not provided a value for ${value}. Please provide a value for ${value} with the --${value} option before continuing`)
+}
+
+function run (cluster, command) {
+  const { alias, oldIndex, newIndex } = command.opts()
+
+  if (!alias) {
+    displayErrorMessage('alias')
+  }
+  if (!oldIndex) {
+    displayErrorMessage('oldIndex')
+  }
+  if (!newIndex) {
+    displayErrorMessage('newIndex')
+  }
```

Comment:
**Suggestion:** I think we need to return on each of these otherwise the command will continue to run after outputting the log.

```suggestion
  if (!alias) {
    return displayErrorMessage('alias')
  }
  if (!oldIndex) {
    return displayErrorMessage('oldIndex')
  }
  if (!newIndex) {
    return displayErrorMessage('newIndex')
  }
```

Reply:
Ahh I see you have done it like this:

```
if (!indexName) {
    throw new Error('The --index option is required')
  }
```

I will do that too!

Reply:
I think either way is fine :) Little things like this which is why we should upgrade to later versions of commander soon so we can use `requiredOption`